tests-admin-stub-server/src/main/kotlin/AdminStubStorage.kt

```diff
@@ -31,6 +31,7 @@ class AdminStubStorage {
                     tests[action.sessionId] = testsInfo + it.tests
                 }
             }
+            else -> {} //Ignore me
```

Comment:
Reason for adding this

Reply:
If I remove the else block, I will receive the error: " 'when' expression must be exhaustive; add the necessary 'is AddSessionData' branch or an 'else' branch instead. "

Reply:
So, I can replace `else` block with additional branch AddSessionData with empty body.